src/styles/Colors.scss

```diff
@@ -5,6 +5,7 @@ $darkBlue: #0a66c2;
 $gold: #e0b172;
 $lightGray: lightgray;
 $gray: gray;
+$linkedinBackgroundMainTheme: #f3f2ef;
```

Comment:
why not to use mainTheme?

Reply:
Because mainTheme it is blue color on welcome page and this color appears on every background page in real linkedin
![image](https://user-images.githubusercontent.com/87152087/128005380-7a3ff610-1095-4a93-a804-4913646dc519.png)

but linkedinBackgroundMainTheme is not the best name if you have an idea then give it a go : )

Reply:
so it could 

welcomeMainTheme
and
linkedinMainTheme